apps/web/src/api/hooks/integrations/use-active-integrations.ts

```diff
@@ -1,8 +1,9 @@
-import { useQuery } from 'react-query';
+import { useQuery } from '@tanstack/react-query';
+
 import { getActiveIntegrations } from '../../integration';
 
 export function useActiveIntegrations() {
-  const { data, isLoading, refetch } = useQuery('notificationsList', getActiveIntegrations);
+  const { data, isLoading, refetch } = useQuery(['notificationsList'], getActiveIntegrations);
```

Comment:
I've run the codemods from the migration guide.
One of the changes they did was that query key should be now always an array